Sources/alloy-codeless-lite-ios/Public/Utility/PluginURLBuilde.swift

```diff
@@ -3,22 +3,23 @@ import SwiftUI
 
 internal struct PluginURLBuilder {
 
-    let baseUrl = "https://alloysdk.alloy.co/?isNext=true&isWebview=true&"
     var apiKey: String
     var journeyToken: String
     var applicationToken: String
     var production: Bool
     var showHeader: Bool
+    var appUrl: String
 
-    init(apiKey: String, journeyToken: String, applicationToken: String, production: Bool, showHeader: Bool) {
+    init(apiKey: String, journeyToken: String, applicationToken: String, production: Bool, showHeader: Bool, appUrl: String) {
         self.apiKey = apiKey
         self.journeyToken = journeyToken
         self.applicationToken = applicationToken
         self.production = production && false
         self.showHeader = showHeader && true
+        self.appUrl = appUrl
     }
 
     public func getPluginURL() -> String {
-        baseUrl  + "journeyApplicationToken=\(applicationToken)&journeyToken=\(journeyToken)&key=\(apiKey)&production=\(production)&showHeader=\(showHeader)"
+        appUrl  + "/?isWebview=true&journeyApplicationToken=\(applicationToken)&journeyToken=\(journeyToken)&key=\(apiKey)&production=\(production)&showHeader=\(showHeader)"
```

Comment:
do we also need `isNext=true` from the baseUrl?

Reply:
no more. thanks for checking